src/rpc/core_rpc_server.cpp

```diff
@@ -435,6 +509,10 @@ namespace cryptonote
   bool core_rpc_server::on_get_indexes(const COMMAND_RPC_GET_TX_GLOBAL_OUTPUTS_INDEXES::request& req, COMMAND_RPC_GET_TX_GLOBAL_OUTPUTS_INDEXES::response& res)
   {
     PERF_TIMER(on_get_indexes);
+    bool ok;
```

Comment:
`ok` or `r`?

Reply:
Because `bool r = ...` is used a few lines below.